### PR TITLE
Update git-commands extension

### PR DESCRIPTION
**Pull request by manumorante**
## Favorites and Recently used

git-commands-1

- Manages a list of favorites that will appear at the beginning.
- List of aliases you have recently used or visited.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Should we include `@raycast/utils` so we can use `useCachedPromise`, allowing us to use isLoading on the list to ensure everything is loaded before showing data 🙂

**Comment by manumorante** [comment]
> Should we include `@raycast/utils` so we can use `useCachedPromise`, allowing us to use isLoading on the list to ensure everything is loaded before showing data 🙂

@pernielsentikaer It is done! Much better this way :)
Thank you

File: extensions/git-commands/src/command-detail.tsx [hunk 1]
```diff
@@ -1,36 +1,95 @@
-import { Detail } from "@raycast/api";
-import { typeColor, typeDescription } from "./utils";
+import { Action, ActionPanel, Color, Detail, Icon } from "@raycast/api";
```
Comment (pernielsentikaer) [inline extensions/git-commands/src/command-detail.tsx:1]:
```suggestion
import { Action, ActionPanel, Color, Detail, Icon, Keyboard } from "@raycast/api";
```

File: extensions/git-commands/src/command-detail.tsx [hunk 2]
```diff
@@ -1,36 +1,95 @@
...
+                title="Remove Favorites"
+                onAction={handleFavorite}
+                shortcut={{ modifiers: ["cmd"], key: "x" }}
+              />
+            ) : (
+              <Action
+                icon={Icon.Star}
+                title="Save Favorite"
+                onAction={handleFavorite}
+                shortcut={{ modifiers: ["cmd"], key: "s" }}
```
Comment (pernielsentikaer) [inline extensions/git-commands/src/command-detail.tsx:65]:
```suggestion
                shortcut={Keyboard.Shortcut.Common.Pin}
```
Maybe use the default shortcut instead?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

I have now tested your extension, and I have some feedback ready for you

- Maybe add a yellow star as accessory instead of an icon since it makes the list "jump" when it's an icon (and skip the accessory from favorites)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
Here is what I'm talking about: only showing an icon for favorites makes everything jump

git-commands 2024-06-05 at 10 35 57

**Comment by manumorante** [comment]
@pernielsentikaer yep :)
This is how it looks now:

git-commands 2024-06-05 at 11 13 26

You can disable the colors in the extension's preferences :)
I will upload the changes soon.

**Comment by pernielsentikaer** [comment]
Is this ready for a review?

**Comment by manumorante** [comment]
@pernielsentikaer yes! :)

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**